### PR TITLE
Documentation for pgcli magic in ipython

### PR DESCRIPTION
**Pull request by j-bennet**
## Description

This adds an IPython section to the README, to document using pgcli from within IPython console.

## Checklist

- ~[ ] I've added this contribution to the `changelog.md`~.
- [X] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -208,6 +208,91 @@ the docker container:
...
+
+    $ ipython
+    Python 2.7.12 (default, Nov  1 2016, 07:58:43)
+    Type "copyright", "credits" or "license" for more information.
+
+    IPython 5.4.1 -- An enhanced Interactive Python.
+    ?         -> Introduction and overview of IPython's features.
+    %quickref -> Quick reference.
+    help      -> Python's own help system.
+    object?   -> Details about 'object', use 'object??' for extra details.
```
Comment (amjith) [inline README.rst]: Remove the startup message.

File: README.rst [hunk 2]
```diff
@@ -208,6 +208,91 @@ the docker container:
...
+Connect to a database and construct a query:
+
+::
+
+    In [2]: %pgcli postgres://someone@localhost:5432/world
+    Connected: someone@world
+    Version: 1.7.0
+    Chat: https://gitter.im/dbcli/pgcli
+    Mail: https://groups.google.com/forum/#!forum/pgcli
+    Home: http://pgcli.com
```
Comment (amjith) [inline README.rst]: Remove the startup message here as well.

**Comment by j-bennet** [comment]
@amjith Fixed.

**Comment by amjith** [comment]
🍳

**Pull request merged**